arch/arm/core/aarch32/cortex_m/Kconfig

```diff
@@ -267,21 +267,18 @@ config DYNAMIC_DIRECT_INTERRUPTS
 
 config SW_VECTOR_RELAY
 	bool "Enable Software Vector Relay"
-	depends on ARMV6_M_ARMV8_M_BASELINE && !(CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP || CPU_CORTEX_M_HAS_VTOR)
 	help
 	  When building bootloader firmware this option adds a
 	  Vector Table relay handler and relay vector table, to
 	  relay interrupts based on a vector table pointer.
-	  This is only
-	  required for Cortex-M0 (or an Armv8-M baseline core) with no hardware
-	  vector table relocation mechanisms or for Cortex-M0+
-	  (or an Armv8-M baseline core) with no VTOR and no other hardware
-	  relocation table mechanisms.
+	  Normally this is only required for Cortex-M0(+) and Armv8-M baseline
```

Comment:
Just write "Cortex-M Baseline CPUs", instead of "Cortex-M0(+) and Armv8-M baseline cores"